.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: Node
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x, 13.x]
```

Comment:
8.x has now reached End Of Life, it reached it on December 31, 2019. I think we should remove it from CI.

Reply:
Done.